### DISCUSSION TITLE
Binding media keys

### DISCUSSION THREAD

**Discussion by mfdorst** [hyprwm/Hyprland#922] [Q&A] [answered]
To get volume controls to work I assume the thing to do is to bind media keys. I tried this
```
bind = XF86AudioRaiseVolume,, exec, pactl set-sink-volume @DEFAULT_SINK@ +5%
```
and got
```
Config error at line 127 (~/.config/hypr/hyprland.conf): Invalid mod, requested mod "XF86AudioRaiseVolume" is not a valid mod.
```
I don't know if the key is called something different, or if this entirely the wrong approach.
How does one set up volume controls in hyprland?

**Comment by fufexan** [comment, marked as answer]
You've put the key in the modifiers place. Put it after the first `,` and it'll work.

**Reply by mfdorst** [reply]
Oh right, that makes sense. Thanks.
